Java/parkservices/src/main/java/com/amazon/randomcutforest/parkservices/preprocessor/Preprocessor.java

```diff
@@ -765,14 +768,17 @@ protected void shiftLeft(double[] array, int baseDimension) {
      */
     protected double normalize(double value, Deviation[] deviation, double factor) {
         double currentFactor = (factor != 0) ? factor : Math.abs(deviation[2].getMean());
-        if (value - deviation[0].getMean() >= 2 * clipFactor * (currentFactor + DEFAULT_NORMALIZATION_PRECISION)) {
+        if (value - deviation[0].getMean() >= NORMALIZATION_SCALING_FACTOR * clipFactor
+                * (currentFactor + DEFAULT_NORMALIZATION_PRECISION)) {
             return clipFactor;
         }
-        if (value - deviation[0].getMean() < -2 * clipFactor * (currentFactor + DEFAULT_NORMALIZATION_PRECISION)) {
+        if (value - deviation[0].getMean() < -NORMALIZATION_SCALING_FACTOR * clipFactor
```

Comment:
Do you want to use <= instead of < here to be consistent with the >= case above?

Reply:
done